libdirections-onboard/src/test/java/com/mapbox/navigation/route/onboard/MapboxOnboardRouterTest.kt

```diff
@@ -61,11 +62,13 @@ class MapboxOnboardRouterTest {
     private val routerResultFailure: RouterResult = mockk(relaxUnitFun = true)
     private val routerOptions: RouteOptions = provideDefaultRouteOptions()
     private val logger: Logger = mockk(relaxUnitFun = true)
+    private val mockSkuTokenProvider = mockk<SkuTokenProvider>(relaxed = true)
 
     @Before
     fun setUp() {
         every { navigator.configureRouter(any(), any()) } returns 0
-        onboardRouter = MapboxOnboardRouter(ACCESS_TOKEN, navigator, MapboxOnboardRouterConfig(TILE_PATH), logger)
+        every { mockSkuTokenProvider.obtainSkuToken() } returns ("102ka34odzf38e3b8f5f1ba42818e94d31090d6479f")
```

Comment:
what happens when obtainSkuToken is empty? silent fails?

Reply:
It's handled downstream by NN but to answer your question the token it's not appended to the URL.

Reply:
if there is no token source, we shall just use the regular URL.
The same happens if it returns an empty token.